lib/cli/scripts/plugins/aae-plugin.ts

```diff
@@ -0,0 +1,69 @@
+import { PluginInterface } from './plugin-model';
+import { logger } from '../logger';
+import { PluginConfiguration } from './plugin-config';
+
+export class ProcessAutomationPlugin {
```

Comment:
can we rename the file into process-automation-check-plugin?

Reply:
Updated